lib/rules/capitalized-comments.js

```diff
@@ -108,13 +108,17 @@ function createRegExpForIgnorePatterns(normalizedOptions) {
 
 module.exports = {
     meta: {
+        type: "style",
```

Comment:
Even though comment tokens are not part of the AST, I can see a similar argument to https://github.com/eslint/eslint/pull/10912/files#r226037811 being made here, since it would change the value of the comment token.

Reply:
Non-blocking - I'm unsure of what the right course of action is here....

Reply:
Yeah, I think I agree with you.